os/arch/arm/src/armv7-r/arm_assert.c

```diff
@@ -91,6 +91,11 @@
 #include "mpu.h"
 #endif
 
+#if defined(CONFIG_FS_ROMFS) && defined(CONFIG_FRAME_POINTER)
+#include <stdio.h>
+bool abort_mode = false;
```

Comment:
To use type bool, you should include stdbool.h

Reply:
Done. I added stdbool.h outside my config as bool type is used by existing code as well